db/migrate/20210226014754_create_authors.rb

```diff
@@ -0,0 +1,22 @@
+class CreateAuthors < ActiveRecord::Migration[5.2]
+  def change
+    create_table :authors do |t|
+      t.string :name
+      t.string :biography
+      t.string :publisher_name
+      t.string :publisher_email
+      t.timestamps
+    end
+
+    add_index :authors, :name, unique: true
+
+    change_table :books do |t|
+      t.belongs_to :author, index: true, foreign_key: true
+    end
```

Comment:
Do you still need this?

Reply:
When I removed it, the app still works. I'm assuming this is only for the has one/has many associations?